Tests/XCDBLDTests/BuildArgumentsSpec.swift

```diff
@@ -3,8 +3,10 @@ import Quick
 import Nimble
 import XCDBLD
 
-class BuildArgumentsSpec: QuickSpec {
-	override func spec() {
+ // swiftlint:disable:this file_length
```

Comment:
Yeah, let's disable file_length in all test classes using their own `.swiftlint.yml` instead.